tests/console/yast2_clone_system.pm

```diff
@@ -24,10 +24,11 @@ sub run() {
     script_run("yast2 clone_system; echo yast2-clone-system-status-\$? > /dev/$serialdev", 0);
 
     # workaround for bsc#1013605
-    assert_screen([qw(dhcp-popup yast2_console-finished)], 200);
+    my $timeout = 600;
+    assert_screen([qw(dhcp-popup yast2_console-finished)], $timeout);
```

Comment:
1. if it's workaround for a bug why not use soft fail ?
2. bug mentioned in comment above is verified and fixed